MODULE.bazel

```diff
@@ -6,3 +6,4 @@ module(
 
 bazel_dep(name = "platforms", version = "0.0.8")
 bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_python", version = "0.29.0")
```

Comment:
In your own module file, add your use_extension() call of your own extension. This will trigger your extension and cause it to be invoked. Users then don't need to do it themselves unless they want a reference to the `@pybind` repo

Reply:
Acknowledged. I'm going to handle all that in the next PR. (When I do so, I'll implement your advice above regarding extensions.)